kaspresso/src/main/kotlin/com/kaspersky/kaspresso/device/screenshots/Screenshots.kt

```diff
@@ -16,6 +16,15 @@ interface Screenshots {
      */
     fun take(tag: String)
 
+    /**
+     * Takes full screenshot if it is possible.
+     *
+     * Required Permissions: WRITE_EXTERNAL_STORAGE.
+     *
+     * @param tag a unique tag to further identify the screenshot. Must match [a-zA-Z0-9_-]+.
+     */
+    fun takeFull(tag: String)
```

Comment:
@Areyana could you help me to know the purpose of these methods? If you want the basic screenshot engine in the test to be able to take fullscreen screenshots, this is a good idea, but then we need to have an additional option in Kaspresso Builder.